packages/node/src/policy-gen/policy-gen-compartment-map.js

```diff
@@ -36,6 +36,24 @@ export const loadCompartmentMap = async (
     },
   })
 
+  // In lavamoat the canonical name is what we present to the end user as a resource key in policy. We have no use for Endo's label field, so we replace labels with canonicalNames derived from path.
+  // NOTE: the algorithm creating paths happens to be identical to the one in @lavamoat/aa package. Not that it matters because policies cannot be reused between this and other lavamoat tools.
```

Comment:
@naugtur

> Not that it matters because policies cannot be reused between this and other lavamoat tools.

I'm not sure I follow; do you mean that a policy created for a node app won't be helpful if you try to bundle that app and run it in the browser?  not that the policy _schema_ is incompatible?